nntrainer/layers/time_dist.h

```diff
@@ -93,6 +93,24 @@ class TimeDistLayer : public Layer {
    */
   std::shared_ptr<Layer> getDistLayer() { return dist_layer; };
 
+  /**
+   * @brief     get transposed Tensor
```

Comment:
Could you add a comment how it is transposed?

Reply:
done